v2/internal/runner/runner.go

```diff
@@ -307,6 +311,24 @@ func (r *Runner) RunEnumeration() error {
 		}
 		r.options.Templates = append(r.options.Templates, templatesLoaded...)
 	}
+	if len(r.options.NewTemplatesWithVersion) > 0 {
+		minVersion, _ := semver.Parse("8.8.5")
```

Comment:
`err` should be checked and propagated if not nil

Reply:
Done